### DISCUSSION TITLE
Is it possible to have two accounts at GitHub and have same mobile for 2FA?

### DISCUSSION THREAD

**Discussion by gsamaras** [community/community#74606] [Other Feature Feedback, Questions, & Ideas]
One account is for a work related project and the other account is personal - can they share the same mobile phone number for SMS codes for 2FA authentication?

Related question: https://www.reddit.com/r/github/comments/15uoco6/is_it_possible_to_have_2_accounts_at_github_and/

---

Story behind this question: I've found myself into a situation where I need to maintain a public project's repo, for which a former colleague had created a dedicated account. Now GitHub says I need to enable 2FA for this project's repo. Can I use the same mobile phone number I am using for my personal GitHub account? I don't want to install any extensions and such for authentication procedures.

**Comment by asb-realrev** [comment]
I have an ignored (3+ weeks) open support cases with Github on mandatory 2FA (with threatened lockouts) since I was able to configure SMS 2FA successfully with one account, but the other second one keeps on failing during Github's enrollment without any details other than it "failed". 
So I'm wondering if that perhaps might be Github's internal bug, with zero helpful failure/error messages nor any documentation of those additional requirements. 

If others already know or are aware of this limitation, PLEASE share!

**Reply by asb-realrev** [reply]
https://github.com/orgs/community/discussions/44020 
Came across this post which suggests that at some point SMS for multiple accounts to the same phone number DID work and was allowed (start of 2023).
